packages/ethereum-contracts/test/foundry/FoundrySuperfluidTester.sol

```diff
@@ -92,6 +92,7 @@ contract FoundrySuperfluidTester is Test {
             _expectedTotalSupply += INIT_SUPER_TOKEN_BALANCE;
             vm.stopPrank();
         }
+        superToken.warmUpCache();
```

Comment:
Why should this be called by test code instead? If

Reply:
not sure where this would be called instead tbf